tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,48 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(9)
```

Comment:
```suggestion
  skip_if_too_deep(5)
```
The depth can only be a number between 1-5. Please change it to 5.

Reply:
my bad, thanks